### PR TITLE
docs: Documentation rewrite

### PR DESCRIPTION
**Pull request by cdrage**
### What does this PR do?

Adds A LOT more information regarding how to use bootc / containerfile /
quadlets / etc.

### Screenshot / video of UI

N/A, preview the markdown.

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop-extension-bootc/issues/66
Closes https://github.com/containers/podman-desktop-extension-bootc/issues/33
Closes https://github.com/containers/podman-desktop-extension-bootc/issues/26
Closes https://github.com/containers/podman-desktop-extension-bootc/issues/25

### How to test this PR?

N/A, preview the markdown.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: CONTAINERFILE_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+```
+
+There are other non-official tools such as [podlet](https://github.com/k9withabone/podlet) that can automatically generate the systemd-compatible file from a `podman run` command which can help generate a file quicker.
+
+2. Create the `Containerfile`:
+
+```Dockerfile
+FROM quay.io/centos-bootc/fedora-bootc:eln
+
+COPY storage.conf /etc/containers/storage.conf
```
Comment (sallyom) [inline CONTAINERFILE_GUIDE.md:176]: Is the storage.conf necessary? Is there an example storage.conf?

Comment (sallyom) [reply CONTAINERFILE_GUIDE.md:176]: nm, found it below :)

Comment (cdrage) [reply CONTAINERFILE_GUIDE.md:176]: Unfortunately yes as of right now :(.. As there is only a few persistent folders for the container OS.

I cannot find the issue, but I believe this "hack" will be changed in the future. Same goes for the whole 'systemctl enable" commands not working within a container.

Comment (sallyom) [reply CONTAINERFILE_GUIDE.md:176]: I have been running bootc images without the storage.conf addition - seems to work fine 🤷

File: docs/vm_guide.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    --device virtio-rng \
+    --device virtio-input,keyboard \
+    --device virtio-input,pointing \
+    --device virtio-gpu,width=1920,height=1080 \
+    --gui
+```
+
+## Windows
+
+TODO
```
Comment (cdrage) [inline docs/vm_guide.md:45]: TODO for now, we can add a PR in the future documenting this 👍

**Review by deboer-tim — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]

File: CONTAINERFILE_GUIDE.md (hunk 1 above)
Comment (sallyom) [reply CONTAINERFILE_GUIDE.md]: fyi - the additional stores in storage.conf is configured by default now in fedora-bootc:eln - however it is not yet in centos-bootc:stream9 - I learned the hard way, ran out of space when pulling images in a centos-bootc system.

**Pull request merged**